runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CoordinateClue.java

```diff
@@ -52,9 +201,23 @@ public void makeOverlayHint(PanelComponent panelComponent, ClueScrollPlugin plug
 	{
 		panelComponent.getChildren().add(TitleComponent.builder().text("Coordinate Clue").build());
 
-		panelComponent.getChildren().add(LineComponent.builder()
-			.left("Click the clue scroll along the edge of your world map to see where you should dig.")
-			.build());
+		String solution = CLUES.get(location);
+
+		if (solution != null)
+		{
+			panelComponent.getChildren().add(LineComponent.builder()
+				.left(solution)
+				.build());
+
+			panelComponent.getChildren().add(LineComponent.builder().left("").build());
```

Comment:
Is this the best way to get an empty line?

Reply:
Apparently it is. We have handling for `<br>` in tooltips, but not in LineComponents or TitleComponents. This is the same as how the clue scroll plugin adds "Spade required" text: https://github.com/runelite/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/MapClue.java#L165-L166